bindings/java/src/main/native/Makefile.am

```diff
@@ -63,7 +63,7 @@ install-data-hook:
 
 #Remove all compiled Java files
 clean-local:
-	$(MVNCMD) clean
+	rm -rf $(topdir)/bindings/java/src/main/native/build-java/
```

Comment:
@petro-rudenko can you use `MOSTLYCLEANFILES`?
https://www.gnu.org/savannah-checkouts/gnu/automake/manual/html_node/Clean.html#Clean